scss/bitstyles/utilities/inset/inset.stories.mdx

```diff
@@ -0,0 +1,84 @@
+import { Canvas, Meta, Story } from '@storybook/addon-docs/blocks';
+
+<Meta title="Utilities/inset" />
```

Comment:
Was just wondering: now that we have a load of utilities, would it make sense to group them further? E.g. `Utilities/Position/inset`?

Reply:
Yes good idea 🤔 There’s probably others that could be grouped too — issue here: https://github.com/bitcrowd/bitstyles/issues/659